ace-window.el

```diff
@@ -402,6 +471,16 @@ selected window).
 Prefixed with two \\[universal-argument]'s, deletes the selected
 window."
   (interactive "p")
+  ;; Signal an error if `aw-make-frame-char' is ever set to an invalid
```

Comment:
This should be handled by `defcustom`. See `ediff-diff-options` - it has a `:set 'ediff-set-diff-options` property that does the checking.

Reply:
Ok, I could do this but I saw you removed this defcustom of aw-make-frame-char in the patches you applied.  Was that an oversight or did you just want this change before applying it?